core/iwasm/common/gc/gc_type.c

```diff
@@ -1188,19 +1197,15 @@ wasm_reftype_is_subtype_of(uint8 type1, const WASMRefType *ref_type1,
                 }
 #endif
                 else if (heap_type == HEAP_TYPE_NONE) {
-                    /* (ref none) */
-                    /* TODO */
-                    bh_assert(0);
+                    return wasm_is_reftype_supers_of_none(type2, NULL, types,
+                                              type_count);
```

Comment:
Coding style: should indent to the column of the first parameter in above line. You can use clang-format to auto format your patch